src/components/ExpandableInfo.tsx

```diff
@@ -1,216 +1,155 @@
-  border-radius: 2px;
-  padding: 1.5rem;
-  display: flex;
-  flex-direction: column;
-  margin-bottom: 1rem;
-  background: ${({ background, theme }) =>
-    background ? theme.colors[background] : theme.colors.background};
-  &:hover {
-    img {
-      transform: scale(1.08);
-      transition: transform 0.1s;
-    }
-  }
-`
-
-const Content = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  gap: 3rem;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    gap: 2rem;
-    flex-direction: column;
-  }
-`
-
-const TitleContent = styled.div`
-  display: flex;
-  align-items: center;
-  gap: 3rem;
-  width: 100%;
-`
-
-const Title = styled.h2`
-  margin-top: 0rem;
-  margin-bottom: 0.5rem;
-`
-
-const TextPreview = styled.p`
-  font-weight: 400;
-  color: ${(props) => props.theme.colors.text200};
-  margin-bottom: 0rem;
-`
-
-const Text = styled(motion.div)`
-  font-size: 16px;
-  font-weight: 400;
-  color: ${(props) => props.theme.colors.text};
-  margin-top: 2rem;
-  border-top: 1px solid ${(props) => props.theme.colors.border};
-  padding-top: 1.5rem;
-`
-
-const Question = styled.div`
-  margin-right: 1rem;
-`
-
-const Header = styled.div`
-  display: flex;
-  width: 100%;
-  margin: 1rem 0;
-  align-items: center;
-  img {
-    margin-right: 1.5rem;
-  }
-`
-
-const ButtonContainer = styled(motion.div)`
-  display: flex;
-  width: 5rem;
-  justify-content: center;
-  align-items: center;
-  min-height: 10rem;
-  cursor: pointer;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    min-height: 100%;
-    height: 100%;
-    width: 100%;
-    margin: 0;
-  }
-  &:hover {
-    svg {
-      transform: scale(1.25);
-      transition: transform 0.1s;
-    }
-  }
-`
-
-export interface IProps {
+export interface IProps extends ChakraProps {
   children?: React.ReactNode
   image?: IGatsbyImageData
   title: ReactNode
   contentPreview: ReactNode
-  background: string
-  forceOpen: boolean
+  background: BackgroundProps["background"]
+  forceOpen?: boolean
   className?: string
 }
 
-const ExpandableInfo: React.FC<IProps> = ({
-  image,
-  title,
-  contentPreview,
-  children,
-  background,
-  forceOpen,
-  className,
-}) => {
-  const [isVisible, setIsVisible] = useState(forceOpen)
-  const expandCollapse = {
-    collapsed: {
-      height: 0,
-      transition: {
-        when: "afterChildren",
-      },
-    },
-    expanded: {
-      height: "100%",
-      transition: {
-        when: "beforeChildren",
-      },
+const ExpandableInfo = forwardRef<IProps, "div">(
+  (
+    {
+      image,
+      title,
+      contentPreview,
+      children,
+      background,
+      forceOpen,
+      className,
+      ...rest
     },
-  }
-  const chevronFlip = {
-    collapsed: {
-      rotate: 0,
-      transition: {
-        duration: 0.1,
+    ref
+  ) => {
+    const { isOpen, getButtonProps, getDisclosureProps } = useDisclosure({
```

Comment:
This is used to provide accessibility for this component, should toggling expansion be used (`forceOpen` = false). `getButtonProps` holds all the necessary `aria-*` attributes for toggling, including `aria-expanded` and `aria-controls`.

`getDisclosureProps` Provides `id` prop for `aria-controls` and the `hidden` attribute, which is needed in conjunction with the accordion pattern from the [WAI-Aria Authoring Practices](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/)

This of course is on contingent on if accordion behavior should still exist.